Cakes-Co/index.html

```diff
@@ -1,16 +1,115 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inria+Sans:wght@700&display=swap"
+      rel="stylesheet"
+    />
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+  </head>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+  <body>
+    <header class="header">
+      <img class="logo" src="images/cake-logo.png" alt="Cakes Co Logo" />
+      <h1 class="title">Cakes Co</h1>
+      <section class="nav-text">
+        <label for="trigger">
```

Comment:
I'm not sure what the intention is behind the `<label>` element, and it also seems to have distorted your indentation on line 30. Make sure you're using Prettier to format all code